pythonforandroid/bootstraps/service_only/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -92,6 +92,10 @@
             </intent-filter>
         </receiver>
         {% endif %}
+
+        {%- if args.content_providers -%}
```

Comment:
Why {%- rather than just {% (and same elsewhere)?

Reply:
No other reason apart from not being familiar with Jinja and copy-pasting the lines that render the intent filters. I will fix it.